packages/menu/src/MenuItem.tsx

```diff
@@ -255,6 +257,12 @@ const MenuItem = React.forwardRef(
         </Root>
       </li>
     );
+
+    if (usesLinkElement({ ...rest })) {
```

Comment:
This is the same as `usesLinkElement(rest)` I believe.

Reply:
I'm also not convinced this is worth an entire function to check `href != null`...

Reply:
If we got rid of that function, we could destructure href from props, and write the declaration for `anchorProps` above like so:
```js
const anchorProps = href && {
  target: `_blank`,
  rel: 'noopener noreferrer',
  href,
}
```

Reply:
Leaving as-is based on our conversation